### PR TITLE
Link to the notebook configuration docs.

### PR DESCRIPTION
**Pull request by rgbkrk**
Link to the notebook config docs in case people don't have a config file yet or need to know how to find it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ To install the `IPython Clusters` tab in Jupyter Notebook, add this to your `jup
 c.NotebookApp.server_extensions.append('ipyparallel.nbextension')
 ```
 
+See the [documentation on configuring the notebook server](http://jupyter-notebook.readthedocs.org/en/latest/examples/Notebook/Configuring%20the%20Notebook%20and%20Server.html)
```
Comment (Carreau) [inline README.md]: oh, not http**s** :-)

Comment (rgbkrk) [inline README.md]: Guh! I just went straight from what was in my URL bar. Didn't even know https was an option. I'd rather that be the link anyway, so the HTTPS ones are the defaults that people get to.

Comment (Carreau) [inline README.md]: > Didn't even know https was an option. I'd rather that be the link anyway, so the HTTPS ones are the defaults that people get to.

No sure if want http or https anymore...

Comment (rgbkrk) [inline README.md]: Wait, really?

Comment (Carreau) [inline README.md]: personally I would have used `gopher://` it's secure, nobody uses it.

**Pull request merged**